guides/common/modules/con_provisioning-management-with-project-additional-resources.adoc

```diff
@@ -2,3 +2,6 @@
 = Additional resources
 
 * For information about how to provision hosts with {Project}, see {ProvisioningDocURL}[_{ProvisioningDocTitle}_].
+ifdef::orcharhino[]
+* For more information about compute resource providers, see xref:sources/compute_resources.adoc[Compute Resources].
```

Comment:
'Resources' with capital R is intentional?

Reply:
Yes. This is a whole other story: orcharhino docs use title case almost everywhere. Original: https://docs.orcharhino.com/or/docs/sources/compute_resources.html